Kconfig

```diff
@@ -36,7 +36,7 @@ config VIBRATOR_PRIORITY
 
 config VIBRATOR_STACKSIZE
 	int "stack size"
-	default 4096
+	default 8192
```

Comment:
let's optimize the stack usage by the stack dynamical analysis tool

Reply:
Okay, the stack analysis tool in the QEMU environment shows that there is room for optimization in vibratord_main.